Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```diff
@@ -1659,3 +1659,19 @@
       requirements:
         MatterBin: 2
         Manipulator: 2
+
+- type: entity
+  id: MoneyPrinterMachineCircuitboard
+  parent: BaseMachineCircuitboard
+  name: counterfeiting station machine board
+  description: A strange circuitboard for an illegal machine. You should probably turn this into Security... but let's be honest. You won't.
+  components:
```

Comment:
do we still have contraband tags? this should use the major contra tag if we do (the machine entity should too)

Reply:
we dont have contra tags